src/application/account/Player.java

```diff
@@ -41,10 +42,21 @@ public BoDCharacter getCharacter()
     /**
      * Creates a new character for the player to control.
      */
-    public void createNewCharacter(int id)
+    public void createNewCharacter(int id, Specializations spec)
     {
-        this._character = new BoDCharacter(id, new Point2D(100 + (int) Math.random() * 900, 100 + (int) Math.random() * 400), 50, 50,
-                200, new Image("images/ball_blue.png")); // TODO image would be dynamic if player has different cosmetics
+        switch(spec) {
+        case BLASTER:
+            this._character = new Blaster(id, new Point2D(100 + (int) Math.random() * 900, 100 + (int) Math.random() * 400), new Image("images/ball_blue.png"));
```

Comment:
please at the top define x and y as variables instead. same goes for the image, and also i thought we had specified a dictionary with images somewhere since its quite expensive to create a new image for each object allocation
`int x = 100 + (int)(Math.random() * 900);`
`int y = 100 + (int)(Math.random() * 400);`